### ISSUE TITLE
[VSCode only] false positive: f-string without placeholders (F541)

### ISSUE THREAD

**Issue by moritzwilksch** [charliermarsh/ruff-vscode#96]
I'm having an issue where the ruff VSCode extension shows a false positive F541 while the ruff CLI behaves correctly.

## MRE
```python
# demo.py
x = 42.123
s = f"{x:.2f}"
```

CLI:
```bash
ruff demo.py  # no complaints
```

VSCode:
![image](https://user-images.githubusercontent.com/58488209/210215159-115f34f1-fe4d-4715-b430-6f3483f0f6f5.png)

## Details
VSCode
```
Version: 1.74.2
Commit: e8a3071ea4344d9d48ef8a4df2c097372b0c5161
Date: 2022-12-20T10:32:21.587Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.2.0
Sandboxed: No
```

ruff-vscode
```
v2022.3.13651804
```

ruff
```
ruff 0.0.206
```

PS: thanks for the great work on ruff, I'm absolutely loving it!

**Comment by charliermarsh** [comment]
Thanks! This should've been fixed in the latest pre-release... let me take a look.

**Comment by charliermarsh** [comment]
Oooh hahaha it's because it's now 2023 🤦 And the pre-release minor version is based on the calendar day of the year.

**Comment by charliermarsh** [comment]
Okay, confirmed this is fixed in the latest pre-release.

**Comment by moritzwilksch** [comment]
Haha nice, ruff's very own Y2K ;) Thanks for fixing it!
Cheers

**Comment by charliermarsh** [comment]
Thanks for reporting, I would've missed this!
